man/tpm2_evictcontrol.1.md

```diff
@@ -74,18 +74,18 @@ within the TPM. If an object is *persisted* then the object is resident at the
 
 ## To make a transient handle persistent
 ```
-tpm2_evictcontrol -a o -c object.context -p 0x81010002 -P abc123
+tpm2_evictcontrol -C o -c object.context -p 0x81010002 -P abc123
```

Comment:
@williamcroberts can 
"-c object.context" become an argument. --> This is the input object worked on
 "-p <persistent-handle>" become "-c <persistent-handle>" --> This child object requires parent (-C) with auth (-P)